lib/apiservers/engine/backends/archive_test.go

```diff
@@ -99,34 +105,137 @@ func TestFindArchiveReaders(t *testing.T) {
 		{
 			containerSourcePath: "/",
 			expectedPrefices:    []string{"/", "/mnt/A", "/mnt/B", "/mnt/A/AB"},
+			expectedFilterSpecs: map[string]ReaderFilters{
+				"/": {
+					rebase:  "",
+					exclude: []string{"/mnt/A", "/mnt/B", "/mnt/A/AB"},
+					include: "",
+				},
+				"/mnt/A": {
+					rebase:  "/mnt/A",
+					exclude: []string{"/mnt/A/AB"},
+					include: "",
+				},
+				"/mnt/B": {
+					rebase:  "/mnt/B",
+					exclude: []string{},
+					include: "",
+				},
+				"/mnt/A/AB": {
+					rebase:  "/mnt/A/AB",
+					exclude: []string{},
+					include: "",
+				},
```

Comment:
That's down to the interpretation - on the portlayer side currently, include wins. There's a need for unit tests to `lib/archive/` to assert this behaviour.